OracleTuxedo/core/samples/shm/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Pull Tuxedo base image
+FROM oracle/tuxedo:12.2.2
+
+MAINTAINER Judy Liu<judy.liu@oracle.com>
+COPY simpapp_runme.sh /u01/oracle/
+
+USER root
+RUN chown oracle:oracle -R /u01/oracle/ && \
+    chmod +x /u01/oracle/simpapp_runme.sh
+
+USER oracle
+WORKDIR /u01/oracle
+
+# Define ENTRYPOINT. 
+ENTRYPOINT ["bash","/u01/oracle/simpapp_runme.sh"]
```

Comment:
Please don't run `bash` directly. Just make sure your script is executable and has the `bash` shebang line at the top.